.travis.yml

```diff
@@ -52,6 +84,18 @@ install:
           conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION;
           source activate test-env;
       fi;
+    - if [[ "$MONTEPYTHON310" == 1 ]];
+      then
+          wget https://github.com/brinckmann/montepython_public/archive/3.1.0.tar.gz;
+          tar -xf 3.1.0.tar.gz;
+          pip install montepython_public-3.1.0/;
+      fi;
+    - if [[ "$MONTEPYTHONMASTER" == 1 ]];
+      then
+          wget https://github.com/brinckmann/montepython_public/tarball/master;
+          tar -xf master;
+          pip install brinckmann-montepython_public-*/;
+      fi;
```

Comment:
Does this look generally sensible?

Where is this executed? Is this done temporarily and then removed or am I missing something and potentially spamming some folder with this...?

Reply:
No, travis will clean everything up on its servers once we're done, so you're fine to leave them as they are, providing they're not in a location that interferes with imports (which I think by default they are not).